third_party/3/pandas/_typing.pyi

```diff
@@ -90,7 +90,12 @@ Function = Union[np.func, Callable[..., Any]]
 # Also including function names e.g. np.exp. 'sqrt'
 FunctionOrName = Union[Function, str]
 # Used in SelectionMixin and inheriting classes
-AggregationFunction = Union[FunctionOrName, List[FunctionOrName], Dict[Axis, Union[FunctionOrName, List[FunctionOrName]]]]
+MultipleAggregationFunctions = Union[
+    None,  # aggregation funcs are being specified via kwargs
+    List[FunctionOrName],
+    Dict[Axis, Union[FunctionOrName, List[FunctionOrName]]],
+]
+AggregationFunction = Union[FunctionOrName, MultipleAggregationFunctions]
```

Comment:
Notice that `AggregationFunction` is also assigned to `TransformFunction` in `49:frame.pyi`